trunk/includes/pardot-api-class.php

```diff
@@ -578,7 +577,8 @@ function get_response( $item_type, $args = array(), $property = 'result', $paged
                     'compress'		=> false,
                     'decompress'	=> true,
                     'sslverify' 	=> false,
-                    'headers'       => $headers
+                    'headers'       => $headers,
+                    'offset' => $paged > 1 ? ($paged - 1) * 200 : 0
```

Comment:
The `wp_remote_post` function won't know what to do with `offset` here. I think we'll need to put `offset` as an element in an array, and then make the key here `body`.

Reply:
Here's a good example for how the `body` argument would be used: https://developer.wordpress.org/reference/functions/wp_remote_post/#comment-622

Reply:
:roger: Now you make me wonder if my plugin cache actually got cleared.